core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -56,6 +55,11 @@ object StorageTool extends Logging {
       formatParser.addArgument("--ignore-formatted", "-g").
         action(storeTrue())
 
+      if (args.length == 0 || args.contains("--help")) {
```

Comment:
Thank you for this PR, kafka-storage.sh is using modern Argparse4J format, so we should also account for "-h" here(which is an abbreviation of --help).

Reply:
@dengziming Thanks for your reminder, this is a good catch.